pkgs/development/python-modules/percol/default.nix

```
@@ -1,25 +1,24 @@
-{ lib, buildPythonPackage, fetchPypi, isPy3k
-, six
+{ lib, buildPythonPackage, fetchFromGitHub,
+  cmigemo
 }:
 
 buildPythonPackage rec {
   pname = "percol";
-  version = "0.2.1";
-  disabled = isPy3k;
+  version = "4b28037";
```

```suggestion
  version = "unstable-XXXX-XX-XX";
```

Replace XXXX-XX-XX with the date of the commit.